### PR TITLE
[origin] Removing unused configuration

### PR DESCRIPTION
**Pull request by vwalek**
Removed the diagnostics part as it is no longer maintained and doesn't
work on Openshift.

Adding additional projects to collect.

Removed getting all namespaces as it is not needed for troubleshooting
and project names are sensitive for some customers.

Adding condition to collect the logs from systemd openshift services if
not running as static pods.

Signed-off-by: Vladislav Walek <22072258+vwalek@users.noreply.github.com>

---

### CODE REVIEW

File: sos/report/plugins/origin.py [hunk 1]
```diff
@@ -40,14 +40,7 @@ class OpenShiftOrigin(Plugin):
...
 
-    option_list = [
-        PluginOpt('diag', default=True,
-                  desc='Collect oc adm diagnostics output'),
-        PluginOpt('diag-prevent', default=True,
-                  desc='Use --prevent-modification with oc adm diagnostics'),
-        PluginOpt('all-namespaces', default=False,
-                  desc='collect dc output for all namespaces')
-    ]
+    option_list = []
```
Comment (pmoravec) [inline sos/report/plugins/origin.py:43]: No need of this line, as `Plugin` class already sets that (https://github.com/sosreport/sos/blob/main/sos/report/plugins/__init__.py#L551).

File: sos/report/plugins/origin.py [hunk 2]
```diff
@@ -145,32 +141,26 @@ def setup(self):
             ]
 
             self.add_cmd_output([
-                '%s get -o json %s' % (oc_cmd_admin, jcmd) for jcmd in jcmds
+                '%s get -o yaml %s' % (oc_cmd_admin, jcmd) for jcmd in jcmds
```
Comment (TurboTurtle) [inline sos/report/plugins/origin.py:144]: Is there anything currently using the json outputs? Anything in Insights or other tooling?


Also, the `postproc()` method will need to be updated to account for not only the file name change but also any corrections to ensure it keeps working.

**Review by TurboTurtle — CHANGES_REQUESTED** [review]

File: sos/report/plugins/origin.py (hunk 1 above)
Comment (vwalek) [reply sos/report/plugins/origin.py:43]: Ok, I will remove that line. Thx

File: sos/report/plugins/origin.py (hunk 2 above)
Comment (vwalek) [reply sos/report/plugins/origin.py:144]: Hey, I will put it back to `json`. I put it as `yaml`, but it should be `json`.

**Review by pmoravec — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**